content/en/logs/log_collection/opentelemetry.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Send Logs from OpenTelemetry to Datadog
+kind: documentation
+---
+
+## Overview
+
+[OpenTelemetry][1] (OTel) is an open source observability framework that provides IT teams with standardized protocols and tools for collecting and routing telemetry data. Created as an incubator project by the [Cloud Native Computing Foundation][2] (CNCF), OTel provides a consistent format for instrumenting, generating, gathering, and exporting application telemetry data—namely metrics, logs, and traces—to monitoring platforms for analysis and insight.
+
+The OpenTelemetry Collector is a vendor-agnostic agent process for collecting and exporting telemetry data emitted by many processes. Datadog has an [Exporter][3] available for the OpenTelemetry Collector which allows you to forward traces, metrics, and logs data from OpenTelemetry to Datadog. 
+
+For collecting logs, Datadog recommends using the Collector’s [filelog receiver][4]. The filelog receiver tails the log files that you specify and then the Datadog Exporter that is set up in the Collector sends the log data to Datadog. 
+
+{{< img src="logs/log_collection/otel_collector_logs.png" alt="A diagram showing the host, container, or application sending data the filelog receiver in the collector and the Datadog Exporter in the collector sending the data to the Datadog backend" style="width:100%;">}}
+
+## Set up the Collector and Datadog Exporter
+
+See [Running the Collector][5] and [Configuring the Datadog Exporter][6].
+
+#### Logs and traces correlation
+
+If you are using the Datadog Exporter to also send OpenTelemetry traces to Datadog, use the `trace_parser` operator to extract the `trace_id` from each trace and add it to the associated logs. Datadog automatically correlates the related logs and traces. See [Connect OpenTelemetry Traces and Logs][7] for more information.
+
+{{< img src="logs/log_collection/logs_traces_correlation.png" alt="The trace panel showing a list of logs correlated with the trace" style="width:70%;">}}
+
+## Configure the logger for your application
+
+Since OpenTelemetry SDKs’ logging functionality is not fully supported yet (see your specific language in the [OpenTelemetry documentation][8] for more information), Datadog recommends using a standard logging library for your application. Follow the language-specific [Log Collection documentation][9] to set up the appropriate logger in your application. Datadog strongly encourages setting up your logging library to output your logs in JSON to avoid the need for [custom parsing rules][10]. 
```

Comment:
```suggestion
Since the OpenTelemetry SDKs’ logging functionality is not fully supported (see your specific language in the [OpenTelemetry documentation][8] for more information), Datadog recommends using a standard logging library for your application. Follow the language-specific [Log Collection documentation][9] to set up the appropriate logger in your application. Datadog strongly encourages setting up your logging library to output your logs in JSON to avoid the need for [custom parsing rules][10]. 
```